module-2/lab-hypothesis-testing/your-code/challenge-2.ipynb

```diff
@@ -17,7 +17,7 @@
   },
```

Comment:
- "First let's obtain the unique values of the pokemon types. These values should be extracted from Type 1 and Type 2 aggregated. Assign the unique values to a variable called unique_types". Other way of doing it in Python.
```Python
types = pokemon['Type 1'].append(pokemon['Type 2']).unique()
unique_types = types.tolist()
```
- "Second we will create a list named pokemon_totals to contain the Total values of each unique type of pokemons". Quick way 
```Python

pokemon_totals = []

for ptype in unique_types:
    if isinstance(ptype, str):
        pokemon_totals.append(pokemon.loc[(pokemon['Type 1']==ptype) | (pokemon['Type 2']==ptype)]['Total'])
len(pokemon_totals)
```
- Take a look on the F-statistic as well as on the p-value.